docs/kpod-build.1.md

```diff
@@ -0,0 +1,101 @@
+% kpod(1) kpod-build - Simple tool to build a container image
+% Tom Sweeney
+# kpod-build "7" "November 2017" "kpod"
+
+## NAME
+kpod-build - Build a container image using a Dockerfile.
+
+## SYNOPSIS
+**kpod** **build** [*options* [...]] [**context**]
+
+## DESCRIPTION
+**kpod build** Builds an image using instructions from one or more Dockerfiles and a specified
```

Comment:
Should mention buildah command in here.

Reply:
Add link to buildah man pages

Reply:
I've mentioned the Buildah.  Did you want an additional link here or is the bottom of the page sufficient?

Reply:
Yes I want you to spell it out like you did in the podman build --help text.